circle.yml

```diff
@@ -82,8 +82,8 @@ fedora_install: &fedora_install
     command: |
         echo keepcache=1 >> /etc/dnf/dnf.conf
         dnf upgrade -y libsolv
-        dnf install -y tree git gcc gcc-c++ meson check-devel libudev-devel libevdev-devel valgrind python3-gobject python3-evdev glib2-devel python3-lxml python3-devel swig libunistring-devel
-
+        dnf install -y tree git gcc gcc-c++ meson check-devel libudev-devel libevdev-devel valgrind python3-gobject python3-evdev glib2-devel python3-lxml python3-devel swig libunistring-devel dbus-daemon
+        sed -i 's/systemd//' /etc/nsswitch.conf
```

Comment:
Why is this?

Reply:
see https://github.com/libratbag/libratbag/commit/c4257802206186b891bea8d055c14b03772a64bf, https://github.com/libratbag/libratbag/pull/701#issuecomment-495420829 has a summary too. Without this, we're just failing with dbus-daemon related timeouts